dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/TextToImageServiceExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.TextToImage;
+
+/// <summary>
+/// Extension methods for <see cref="ITextToImageService"/>.
+/// </summary>
+public static class TextToImageServiceExtensions
+{
+    /// <summary>
+    /// Generate an image matching the given description
+    /// </summary>
+    /// <param name="service">Target <see cref="ITextToImageService"/> instance</param>
+    /// <param name="description">Image generation prompt</param>
+    /// <param name="width">Image width in pixels</param>
+    /// <param name="height">Image height in pixels</param>
+    /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <returns>Generated image in base64 format or image URL</returns>
+    [Obsolete("This method is obsolete. Use GetImageContentsAsync instead.")]
```

Comment:
If we go straight to obsolete this could cause builds to fail. Is there any problem just keeping this as a convenience?